signer/core/cliui.go

```diff
@@ -245,6 +245,9 @@ func (ui *CommandlineUI) OnApprovedTx(tx ethapi.SignTransactionResult) {
 
 func (ui *CommandlineUI) showAccounts() {
 	accounts, err := ui.api.ListAccounts(context.Background())
+	var limit int = 20 //max N accounts to print
+	var msg string = fmt.Sprintf("\nFirst %d accounts listed (%d more available).\n", limit, len(accounts)-limit)
```

Comment:
Seems a bit strange that the "you have more than 20 accounts" is the default, which you initialize here. Feels more natural to have the empty-case be the default, `var msg string`, and further below have the clause

``` 
if len(accounts) > limit { 
     msg = fmt.Sprintf("...
```
You can probably simplify that a bit too, e.g. by just shortening the `accounts` to be `llimit` long. And then the code outside of that clause doesn't need to know that a `limit` even exists